sopel/modules/ip.py

```diff
@@ -8,31 +8,23 @@
 https://sopel.chat
 """
 
-from __future__ import absolute_import, division, print_function, unicode_literals
+from __future__ import generator_stop
 
 import logging
 import os
 import socket
 import tarfile
+# urlretrieve has been put under urllib.request in Python 3.
+# It's also deprecated so this should probably be replaced with
+# urllib2.
```

Comment:
```suggestion
```

This comment is somewhat mistaken. `urllib.request.urlretrieve()` is part of the Legacy Interface, and "might become" deprecated in the future but still is not marked so [even in Python 3.10](https://docs.python.org/3.10/library/urllib.request.html#urllib.request.urlretrieve) (so far). There's also no such thing as a `urllib2` module in Python 3.9 or 3.10 (I looked).

Might as well remove the misleading stuff while we're already in here cleaning up.